ci-operator/step-registry/ipi/conf/external-lb/ipi-conf-external-lb-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: ipi-conf-external-lb
```

Comment:
Seem like this step is doing a post-action to configure an external lb for a cluster, but not configure install-config, so suggest to rename it to `vsphere-external-lb`